nifi-nar-bundles/nifi-mqtt-bundle/nifi-mqtt-processors/pom.xml

```diff
@@ -93,5 +93,11 @@
             <artifactId>nifi-schema-registry-service-api</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish</groupId>
+            <artifactId>javax.json</artifactId>
+            <version>1.0.4</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
We more commonly use Jackson for Json processing in the NiFi code base. Would it be possible to use this library here too?

Reply:
Although the `javax.json` library is used in a couple places, I concur with @turcsanyip that we should use Jackson as much as possible. Some third-party libraries depend on alternative JSON implementations, but from a direct dependency management and code maintenance perspective, it is better to use Jackson.

Reply:
After all it doesn't look as bad with Jackson as I thought. I didn't remember that it is possible to chain `.put()`. Changed.